test/AutofacSerilogIntegration.Tests/SourceContext/Scenarios/AcceptsLogViaCtor.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Serilog;
+
+namespace AutofacSerilogIntegration.Tests.SourceContext.Scenarios
+{
+    class AcceptsLogViaCtor : IAcceptsLogViaCtor
+    {
+        readonly ILogger _log;
+
+        public AcceptsLogViaCtor(ILogger log)
+        {
+            _log = log ?? throw new ArgumentNullException(nameof(log));
+        }
+
+        public void CreateLog()
+        {
+            _log.Information("Hello!");
+        }
+    }
+
+    internal interface IAcceptsLogViaCtor : ILogScenario
```

Comment:
Nit: the project doesn't use explicit private/internal modifiers.

Reply:
will fix and pr later.

Reply:
No rush, totally minor, just passing on a heads-up. Thanks for all the help!